packages/jellyfish-api-core/__tests__/category/masternode/setGovHeight.test.ts

```diff
@@ -0,0 +1,115 @@
+import { RpcApiError } from '@defichain/jellyfish-api-core'
+import { GenesisKeys, LoanMasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { createPoolPair, createToken } from '@defichain/testing'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+
+describe('Masternode', () => {
+  let container!: LoanMasterNodeRegTestContainer
+  let client!: ContainerAdapterClient
+
+  beforeEach(async () => {
+    container = new LoanMasterNodeRegTestContainer()
+    client = new ContainerAdapterClient(container)
+
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+
+    const govVar = await client.masternode.getGov('LP_SPLITS')
+    expect(govVar.length).toStrictEqual(1)
+    expect(Object.keys(govVar[0]).length).toStrictEqual(1)
+    expect(Object.keys(govVar[0].LP_SPLITS).length).toStrictEqual(0)
+
+    await createToken(container, 'CAT')
+    await createToken(container, 'DOG')
+    await createPoolPair(container, 'CAT', 'DFI')
+    await createPoolPair(container, 'DOG', 'DFI')
+
+    // to fix each test starting value
+    await client.masternode.setGov({ LP_SPLITS: { 3: 0.2, 4: 0.8 } })
+    await container.generate(1)
+
+    {
+      const govVar = await client.masternode.getGov('LP_SPLITS')
+      expect(govVar.length).toStrictEqual(1)
+      expect(Object.keys(govVar[0]).length).toStrictEqual(1)
+      expect(Object.keys(govVar[0].LP_SPLITS).length).toStrictEqual(2)
+
+      expect(govVar[0].LP_SPLITS['3'].toString()).toStrictEqual('0.2')
+      expect(govVar[0].LP_SPLITS['4'].toString()).toStrictEqual('0.8')
+    }
+  })
+
+  afterEach(async () => {
+    await container.stop()
+  })
+
+  it('should setGovHeight', async () => {
```

Comment:
can add a negative test for not registered gov variable.
Ref https://github.com/DeFiCh/ain/blob/526e71151ceac1efbf305a57d6d15eeff64a8878/src/masternodes/mn_rpc.cpp#L581

Reply:
can add this test in packages/jellyfish-api-core/__tests__/category/masternode/setGov.test.ts also